ir-engine/ir-engine
# IR-4343 Ambient light culling issue

## Summary
adding useEffect along with useOptionalComponent to remove FrustumCullCameraComponent from AmbientLightComponent when it exists. Ambient lights should not be culled based on being within the frustum


## References
[https://tsu.atlassian.net/browse/IR-4343](https://tsu.atlassian.net/browse/IR-4343)

## QA Steps
Create a new scene

Add Ambient Light to the scene

Change the Color to Red

Change intensity from 1 to 10

Change the ambient lighting position to 100,100,100

Observe lighting effect still active in scene
